### PR TITLE
Configurations troubleshooting section

### PR DESCRIPTION
**Pull request by ArashDai**

### CODE REVIEW

**Comment by adepojubosun** [comment]
Great, 
Post rawgit link and link from master branch in comment. 
follow the instructions [here](http://open-learning-exchange.github.io/#!pages/vi/vi-github-issues.md). In the section named "Raw Git"

**Comment by ArashDai** [comment]
Is this correct format for the rawgit link?
https://rawgit.com/ArashDai/ArashDai.github.io/master/#!pages/vi/vi-configurations.md

File: pages/vi/vi-configurations.md [hunk 1]
```diff
@@ -59,5 +59,8 @@ After clicking on the OK button, you will be automatically logged in.
 
 Then, post to the Gitter chat the screenshot you took earlier.
 
-#### Return to [First Steps](vi-first-steps.md#Step_1_-_BeLL_Installation)
+## Troubleshooting
 
+1. When trying to access http://127.0.0.1:5985/apps/_design/bell/MyApp/index.html you may experience an error such as the following: "no_db_found". A simple solution will be using ```vagrant destroy``` to delete the current machine, then try ```vagrant up``` to rebuild it.
```
Comment (Jackbui96) [inline pages/vi/vi-configurations.md]: Shouldn't it be: "A simple solution is to use ```vagrant destroy``` to delete .....". Not "A simple solution will be ..."

Comment (Jackbui96) [reply pages/vi/vi-configurations.md]: Another issue is your Rawgit link is wrong. You are using your master branch for your rawgit preview. The correct branch here should be the alternative one you are using to fix this problem. Please be more careful next time.

Comment (ArashDai) [reply pages/vi/vi-configurations.md]: Regarding the first comment, I literally copied the troubleshooting 2 from step 2 vagrant tutorial, I didn't change the wording. Thanks for the rawgit advice, I've never used it before so im still figuring out how to use it properly

Comment (ArashDai) [reply pages/vi/vi-configurations.md]: so this is the correct rawgit link then: https://rawgit.com/ArashDai/ArashDai.github.io/configurations_troubleshooting_section/#!pages/vi/vi-configurations.md

Comment (Jackbui96) [reply pages/vi/vi-configurations.md]: Yes, that would be the correct format.

**Pull request merged**